.github/scripts/run_mypy.sh

```diff
@@ -3,7 +3,10 @@
 
 set -o errexit
 
+scriptDir=$(realpath ${BASH_SOURCE[0]} --canonicalize-existing)
+rootDir=${scriptDir%/*/*/*}
+
 # change directory to the project root directory.
-cd "$(dirname "$0")"
+cd $rootDir
```

Comment:
why is this change necessary? and what does it do exactly?

Reply:
It reads the script's location and goes up 3 times to reach the project dir.

`BASH_SOURCE[0]` should have the same value as `$0`, but I read that `BASH_SOURCE[0]` is more likely to always produce the wanted result (which is the script's parent directory)

`realpath` turns it from a relative to an absolute path

finally `${scriptDir%/*/*/*}` is [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion) that cuts off the string such that we end up with the path 3 times up. I could have probably written `cd ../../..` instead of that/

Reply:
I forgot to mention, I moved the script, that's the reason I had to change the contents.

Reply:
hm, I am not sure if moving the script inside the github directory makes sense, since it is meant to be used locally as well as for the CI.

Reply:
I just changed the [path](https://github.com/bakdata/kpops/pull/197/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9L36)

Reply:
🤔 I think I get it, it shouldn't be in a github-specific folder, right? Should I move them to a dir outside or do you prefer for me to just leave them in the root directory